### PR TITLE
Fix node support matrix

### PR DESCRIPTION
**Pull request by runspired**

### CODE REVIEW

**Comment by btecu** [comment]
Looks like `ember-cli` along with `ember-try` needs to be bumped to, because it's using `node 8` for testing.

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "qunit-dom": "^0.9.0"
   },
   "engines": {
-    "node": "8.* || 10.* || 12.*"
+    "node": "10.* || >= 12.*"
```
Comment (runspired) [inline package.json:57]:
```suggestion
    "node": "8.* || 10.* || >= 12.*"
```

**Comment by runspired** [comment]
@btecu I restored 8 for now. Would be nice to get this released, prevents us from bumping node version for ember-data :)

**Comment by btecu** [comment]
@runspired I've released version `0.8.0`.

**Pull request closed without merging**